js/ParticipantTopDonor.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
+    // UpdateInfo immediately, and then once every 10 seconds.
+    UpdateInfo();
+    window.setInterval(UpdateInfo, 10000);
+});
+
+function GetTopDonor(data) {    
+    var groupedData = {};
+    $.each(data, function(i, val) {
+        var name = val.donorName;
+        var amount = val.donationAmount;
+        if (groupedData.hasOwnProperty(name)) {
+            groupedData[name] += amount;
+        } else {
+            groupedData[name] = amount;
+        }
+    });
+
+    var sortedData = [];
```

Comment:
if we want to only return 1 person, then sorting is extraneous. but i assume this is for future non-N=1 stuff